README.md

```diff
@@ -40,16 +40,49 @@ The easiest way to find that entity name is by opening the Search within Home As
 
 (The format is, for example: `event.octopus_energy_electricity_{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_day_rates`)
 
-Here's an example yaml configuration - obviously replacing `<your_id_here>` with your data from above. 
+Example configs: - obviously replacing `<your_id_here>` with your data from above. 
+
+Here's an example yaml configuration for import rates:
```

Comment:
I think it might be time to split this in to a simple config example for those jumping on board, and a section under a new header for "Advanced Configurations" - that way folks can see the minimum required to get working versus a more advanced example (with your screenshots)